apps/web-tools/pages/_app.tsx

```diff
@@ -0,0 +1,7 @@
+import "./globals.css";
+import type { AppProps } from "next/app";
+
+export default function MyApp({ Component, pageProps }: AppProps) {
+  // @ts-expect-error FIXME something wrong with typing here
```

Comment:
I didn't figure this out but this code, suggested by the doc, have some type issue. I suspect it is due to React 17 vs 18 typing coercing issue. but i couldn't really prove it.